tests/splines/cosine_evaluator.hpp

```diff
@@ -23,18 +23,18 @@ struct CosineEvaluator
         static constexpr double s_pi_2 = Kokkos::numbers::pi / 2;
 
     private:
-        double m_c0;
+        double m_coef0;
 
-        double m_c1;
+        double m_coef1;
 
     public:
         template <class Domain>
-        explicit Evaluator([[maybe_unused]] Domain domain) : m_c0(1.0)
-                                                           , m_c1(0.0)
+        explicit Evaluator([[maybe_unused]] Domain domain) : m_coef0(1.0)
```

Comment:
```suggestion
        explicit Evaluator(Domain /*domain*/) : m_coef0(1.0)
```

I do not remember exactly, but is there a clang-tidy check for unused variable?